scripts/build_web_vault.sh

```diff
@@ -30,7 +30,7 @@ npm run dist:oss:selfhost
 
 # Create vw-version.json with the latest tag from the remote repo.
 printf '{"version":"%s"}' \
-      "$(git -c 'versionsort.suffix=-' ls-remote --tags --sort='v:refname' https://github.com/dani-garcia/bw_web_builds.git 'v*' | tail -n1 | sed -E 's#.*?refs/tags/v##')" \
+      "$(git -c 'versionsort.suffix=-' ls-remote --tags --sort='v:refname' https://github.com/dani-garcia/bw_web_builds.git 'v*' | tail -n1 | grep -Eo '[^\/]*$')" \
```

Comment:
Unfortunately this introduced issue https://github.com/dani-garcia/bw_web_builds/issues/115

Reply:
Ah, the v is still there. While not a very big issue that should be fixed indeed.

Reply:
yep, see #116